soundpool/pubspec.yaml

```diff
@@ -4,18 +4,21 @@ version: 1.1.3
 homepage: https://github.com/ukasz123/soundpool/tree/master/soundpool
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.10.0 <2.0.0"
+  sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
   flutter:
     sdk: flutter
 
-  soundpool_platform_interface: ^1.0.1
+  soundpool_platform_interface: 
+    path : ../soundpool_platform_interface
```

Comment:
This is fine during development but it cannot reach pub. New versions `2.0.0-nullsafety.0` should be set everywhere.

I already committed version changes